src/drivers/power_monitor/ina226/ina226_main.cpp

```diff
@@ -0,0 +1,298 @@
+		goto fail;
+
+	} else {
+		PX4_INFO("Started INA226 on bus %d", i2c_bus);
+	}
+
+	return PX4_OK;
+
+fail:
+
+	if (g_dev[idx] != nullptr) {
+		delete g_dev[idx];
+		g_dev[idx] = nullptr;
+
+	}
+
+	return PX4_ERROR;
+}
+
+/**
+ * Stop the driver
+ */
+int
+stop(int bus, int address)
+{
+	int idx = get_index(bus, address);
+
+	if (g_dev[idx] != nullptr) {
+		delete g_dev[idx];
+		g_dev[idx] = nullptr;
+
+	} else {
+		PX4_ERR("Driver not running on bus %d, address 0x%02X", bus, address);
+		return PX4_ERROR;
+	}
+
+	return PX4_OK;
+}
+
+/**
+ * Print a little info about the driver.
+ */
+int
+info(int bus, int address)
+{
+	bool any_running = false;
+
+	for (int i = 0; i < MAX_I2C_BATTERY_COUNT; i++) {
+		if (g_dev[i] != nullptr) {
+			any_running = true;
+			PX4_INFO("Bus %d, address 0x%02X:", (int) g_dev[i]->index, g_dev[i]->get_device_address());
+			g_dev[i]->print_info();
+		}
+	}
+
+	if (!any_running) {
+		PX4_INFO("No devices are running.");
+	}
+
+	return PX4_OK;
+}
+
+} /* namespace */
+
+
+static void
+ina2262_usage()
+{
+	PRINT_MODULE_DESCRIPTION(
+		R"DESCR_STR(
+### Description
+Driver for the INA226 power monitor.
+
+Multiple instances of this driver can run simultaneously, if each instance has a separate bus OR I2C address.
+
+For example, one instance can run on Bus 2, address 0x41, and one can run on Bus 2, address 0x43.
+
+If the INA226 module is not powered, then by default, initialization of the driver will fail. To change this, use
+the -f flag. If this flag is set, then if initialization fails, the driver will keep trying to initialize again
+every 0.5 seconds. With this flag set, you can plug in a battery after the driver starts, and it will work. Without
+this flag set, the battery must be plugged in before starting the driver.
+
+)DESCR_STR");
+
+	PRINT_MODULE_USAGE_NAME("ina226", "driver");
+
+	PRINT_MODULE_USAGE_COMMAND_DESCR("start", "Start a new instance of the driver");
+	PRINT_MODULE_USAGE_PARAM_FLAG('a', "If set, try to start the driver on each availabe I2C bus until a module is found", true);
+	PRINT_MODULE_USAGE_PARAM_FLAG('f', "If initialization fails, keep retrying periodically. Ignored if the -a flag is set. See full driver documentation for more info", true);
+	PRINT_MODULE_USAGE_PARAM_INT('b', 0, 0, NUM_I2C_BUS_OPTIONS - 1, "I2C bus (default: use board-specific bus)", true);
+	// The module documentation system INSISTS on decimal literals here. So we can't use a #define constant, and
+	// we can't use hexadecimal.
+	PRINT_MODULE_USAGE_PARAM_INT('d', 65, 0, UINT8_MAX, "I2C Address (in hexadecimal)", true);
+	PRINT_MODULE_USAGE_PARAM_INT('t', 1, 1, 2, "Which battery calibration values should be used (1 or 2)", true);
+
+	PRINT_MODULE_USAGE_COMMAND_DESCR("stop", "Stop one instance of the driver");
+	PRINT_MODULE_USAGE_PARAM_INT('b', 0, 0, NUM_I2C_BUS_OPTIONS - 1, "I2C bus (default: use board-specific bus)", true);
+	PRINT_MODULE_USAGE_PARAM_INT('d', 65, 0, UINT8_MAX, "I2C Address (in hexadecimal)", true);
```

Comment:
Instead of forcing hex, I'd use `strtol(myoptarg, nullptr, 0);`, then you can pass '0x12'